scripts/build.ts

```diff
@@ -88,25 +98,60 @@ export async function build() {
   await fs.ensureDir(OUT_DIR)
   await Promise.all([
     copyPackageJson(),
-    copy("index.d.ts", OUT_DIR),
     copy("README.md", OUT_DIR),
     copy("CHANGELOG.md", OUT_DIR),
     copy("LICENSE", OUT_DIR),
-    fs.writeFile(resolve(OUT_DIR, "jsx-dev-runtime.js"), 'export * from "./jsx-runtime"'),
+    fs.writeFile(resolve(OUT_DIR, "jsx-dev-runtime.js"), 'export * from "./esm/jsx-runtime"'),
```

Comment:
I don't know what's the function of `jsx-dev-runtime.js`, but I try to make it works as previous.